lib/convection/control/stack.rb

```diff
@@ -105,7 +105,7 @@ def []=(key, value)
       end
 
       def get(*args)
-        @attributes.get(*args)
+        @attributes.get(*args, "** Warning: Could not resolve value [#{args[1]}] for stack [#{args[0]}] **")
```

Comment:
I think we should throw an exception. That should surface a stack trace, and give the user an idea of where in the template the null value is getting generated:

```ruby
@attributes.get(*args) { fail, SomeException, 'Attempted to reference an undefined attribute!' }